### ISSUE TITLE
VLAN Interfaces Have MTU 1500

### ISSUE THREAD

**Issue by tahmed-dev** [Azure/sonic-buildimage#5201]
**Description**
VLAN interfaces on host, have default MTU1500. Attempt to reconfigure the MTU to 9100 (Jambo frames). Subsequently, radv does throw a warning message when it add a link MTU:
```
Aug 10 06:38:14.914860 <sonic-hostname> WARNING radv#radvd[44]: AdvLinkMTU for Vlan2820 (9100) must be zero or between 1280 and 1500  
```

**Steps to reproduce the issue:**
1. Reboot into SONiC device with Vlans configured (ToRRouter)
2. Observe Vlan<x> MTU using `sudo ifconfig Vlan<x>`
3. Observe Bridge MTU using `sudo ifconfig Bridge`

**Describe the results you received:**
Vlan<x> MTU is set to 1500
Bridge MTU is set to 1500

**Describe the results you expected:**
Vlan<x> MTU 9100 in order to be inline with member ports MTU

**Comment by tahmed-dev** [comment]
@rlhui, This issue has been fixed via [PR:1393](https://github.com/Azure/sonic-swss/pull/1393)

**Comment by tahmed-dev** [comment]
This issue was opened to track updates into 201911. it is resolved as per above comment. assigning to @rlhui

**Comment by prsunny** [comment]
This is a limitation with 4.9 kernel (201911). Hence the fix is not taken to 201911 but only available in master (4.19 kernel). Please find the notes here - https://github.com/Azure/sonic-swss/pull/1393#issuecomment-679302109
